internal/pkg/cli/init.go

```diff
@@ -251,13 +229,8 @@ func (opts *InitOpts) deploy() error {
 			return err
 		}
 		if !opts.ShouldDeploy {
-			log.Infoln()
-			log.Infoln("No problem, you can deploy your application later:")
-			log.Infof("- Run %s to create your staging environment.\n",
-				color.HighlightCode(fmt.Sprintf("archer env init --name %s --project %s", defaultEnvironmentName, opts.Project)))
-			log.Infof("- Run %s to deploy your application to the environment.\n",
-				color.HighlightCode(fmt.Sprintf("archer app deploy --name %s --env %s --project %s", opts.AppName, defaultEnvironmentName, opts.Project)))
-			log.Infoln()
+			log.Info("\nNo problem, you can deploy your application later:\n")
+			opts.initApp.LogRecommendedActions()
```

Comment:
Maybe it'll be best if we can call the different commands recommended actions and they return a list of actions. So in this case we can print app init and env next steps.

Reply:
Done! It's neater now :)